src/index.js

```diff
@@ -156,6 +160,28 @@ class TemplateContentScript extends ContentScript {
       this.log('info', `Found ${foundContractsNumber} contracts`)
       numberOfContracts = foundContractsNumber
       await this.runInWorker('selectContract', 0)
+      const isError = await this.evaluateInWorker(function checkErrorPage() {
```

Comment:
This is a case where you should declare the checkErrorPage worker methode and call this.runInWorker('checkErrorPage')